File: NotificationSystem/NotificationSystemWeb/Controls_Search/ctrSearch_Agent_Find.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace NotificationSystem.NotificationSystem.Web.Controls_Search
{
	public partial class ctrSearch_Agent_Find : System.Web.UI.UserControl
	{

        private int m_AgentID = 0;

        public int AgentID
        {
            get
            {
                return m_AgentID;
            }
            set
            {
                m_AgentID = value;
            }
        }

        protected void Page_Load(object sender, EventArgs e)
		{

		}

        public void PopulateSearchControl()
		{
            m_AgentID = int.Parse(txtAgentName.Text);
		}
	}
}

File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_Agent_Find.ascx.vb
Imports System
Imports System.Web.UI

Namespace NotificationSystem.NotificationSystem.Web.Controls_Search
    Public Partial Class ctrSearch_Agent_Find
        Inherits UserControl

        Private m_AgentID As Integer = 0

        Public Property AgentID As Integer
            Get
                Return m_AgentID
            End Get
            Set(value As Integer)
                m_AgentID = value
            End Set
        End Property

        Protected Sub Page_Load(sender As Object, e As EventArgs)

        End Sub

        Public Sub PopulateSearchControl()
            m_AgentID = Integer.Parse(txtAgentName.Text)
        End Sub
    End Class
End Namespace
